ecs-agent/tmds/logging/logging_handler.go

```diff
@@ -0,0 +1,34 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package logging
+
+import (
+	"net/http"
+
+	"github.com/cihub/seelog"
+)
+
+// LoggingHandler is used to log all requests for an endpoint.
```

Comment:
This is taken from agent module.

https://github.com/aws/amazon-ecs-agent/blob/0202ad238404be6c4674e8556db2a3319498dbc2/agent/handlers/logging_handler.go#L22-L34